src/templates/_includes/forms/selectize.twig

```diff
@@ -104,6 +105,17 @@
 
     const $select = $(`#${id}`);
 
+    const allowAutocomplete = () => {
+        const $originalSelectInput = $($select[0]);
```

Comment:
@gcamacho079 What’s the point of this line, vs. just referring to `$select` directly in this function?

Reply:
@i-just, could you chime in on this? I’m not too familiar with this change.

Reply:
I tried a few thing before ending up with this code and that's a silly left over, sorry! I pushed an update.